_tools/check_docs.ts

```diff
@@ -586,16 +490,10 @@ if (!lintStatus.success) {
   Deno.exit(1);
 }
 
-await Promise.all(ENTRY_POINT_URLS.map(checkDocs));
-
-const iter = pooledMap(
-  navigator.hardwareConcurrency,
-  snippetPromises,
-  (fn) => fn(),
-);
-for await (const _ of iter) {
-  // noop
+for (const url of ENTRY_POINT_URLS) {
+  await checkDocs(url);
```

Comment:
I think we should keep the current way of checking docs concurrently. This checks items one by one

Reply:
I found the way this was written not very readable and I did some testing, seems like there is little to no impact: 
```sh
file:///std/_tools/check_docs.ts

benchmark   time/iter (avg)        iter/s      (min … max)           p75      p99     p995
----------- ----------------------------- --------------------- --------------------------
pool                  2.1 s           0.5 (   2.1 s …    2.1 s)    2.1 s    2.1 s    2.1 s
loop                  2.1 s           0.5 (   2.1 s …    2.2 s)    2.2 s    2.2 s    2.2 s
```

Reply:
Ok. It seems fine as is